source/reference/kubectl-minio-plugin/kubectl-minio-init.rst

```diff
@@ -94,23 +94,17 @@ The command supports the following flags:
 
    Disabled by default.
 
-.. mc-cmd:: --default-console-image
-   :optional:
-
-   The default :minio-git:`MinIO Console <console>` image to use when creating a new MinIO tenant. 
-   Defaults to ``minio/console:v0.17.3``.
-
 .. mc-cmd:: --default-kes-image
    :optional:
 
    The default :minio-git:`kes <kes>` image to use when creating a new MinIO tenant. 
-   Defaults to ``minio/kes:v0.18.0``.
+   Defaults to the :minio-git:`latest release of KES <kes/releases/latest>`.
```

Comment:
Same comment - e.g. https://github.com/minio/operator/blob/v5.0.4/kubectl-minio/cmd/helpers/constants.go

So it's not guaranteed to be current latest, just whatever is associated to that release.

If this file is generally the right place to look, we could cheat a little bit in the conf and auto-inject the latest release tag to make it easier to link.

Otherwise, it's enough to modify similar to suggestion before, e.g. "defaults to the version bundled in the matching Operator release. See <URL> for more information"

Reply:
right, default value will always be whatever is on the constants.go file, on this case the `DefaultKESImage` env variable.

do you have an example on how could I auto-inject the value from a file?

Reply:
I have to play with this a bit - I'm not sure if there's an easy way to pull and parse that file.

Right now in Makefile we have a dependency builder that snags stuff with wget and parses it from there. 

it would be nicer if this was a YAML of all of our defaults/constants, but that is for later.

For right now we can maybe say...

```
Defaults to the `version bundled in the matching Operator release <https://github.com/minio/operator/blob/master/kubectl-minio//cmd/helpers/constants.go>`__
```

Reply:
applied, and mentioned the variable that holds the value